### PR TITLE
feat: add doucmentation on injecting a model with a named connection

### PR DESCRIPTION
**Pull request by jmcdo29**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Docs
- [ ] Other... Please describe:

## What is the current behavior?

We don't mention the use of `@InjectModel()` with a connection name

Issue Number: https://github.com/nestjs/mongoose/issues/1460

## What is the new behavior?

We mention that with a named connection and `@InjectModel()` you need to specify the connection name

## Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by Tony133** [comment]
Maybe this PR is very similar? http://github.com/nestjs/docs.nestjs.com/pull/2386

**Review by Tony133 — APPROVED** [review]

**Comment by Tony133** [comment]
> > Maybe this PR: [http://github.com/nestjs/docs.nestjs.com/pull/2386](https://github.com/nestjs/docs.nestjs.com/pull/2386) is very similar?
> 
> Oh, sure enough, I was just making a PR after an issue that arose this morning

No problem 👍, maybe we could keep both of them ?

File: content/techniques/mongo.md [hunk 1]
```diff
@@ -253,6 +253,34 @@ To inject a given `Connection` to a custom provider (for example, factory provid
 }
 ```
 
+If you are just looking to inject the model from a named database, you can use the connection name as a second parameter to the `@InejctModel()` decorator.
```
Comment (kamilmysliwiec) [inline content/techniques/mongo.md:256]:
```suggestion
If you are just looking to inject the model from a named database, you can use the connection name as a second parameter to the `@InjectModel()` decorator.
```

File: content/techniques/mongo.md [hunk 2]
```diff
@@ -253,6 +253,34 @@ To inject a given `Connection` to a custom provider (for example, factory provid
...
 
+If you are just looking to inject the model from a named database, you can use the connection name as a second parameter to the `@InjectModel()` decorator.
+
+```typescript
+@@filename(cats.service)
+import { Model } from 'mongoose';
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Cat, CatDocument } from './schemas/cat.schema';
+
```
Comment (kamilmysliwiec) [inline content/techniques/mongo.md:264]:
```suggestion
```

File: content/techniques/mongo.md [hunk 3]
```diff
@@ -253,6 +253,29 @@ To inject a given `Connection` to a custom provider (for example, factory provid
...
+@Injectable()
+export class CatsService {
+  constructor(@InjectModel(Cat.name, 'cats') private catModel: Model<CatDocument>) {}
+}
+@@switch
+import { Model } from 'mongoose';
+import { Injectable, Dependencies } from '@nestjs/common';
+import { getModelToken } from '@nestjs/mongoose';
+import { Cat } from './schemas/cat.schema';
+
```
Comment (kamilmysliwiec) [inline content/techniques/mongo.md:269]:
```suggestion
```

**Pull request merged**